### DISCUSSION TITLE
shifting the whole displayed difficulty range in song select

### DISCUSSION THREAD

**Discussion by iSimpFor** [ppy/osu#19122] [Ideas]
![range](https://user-images.githubusercontent.com/44514667/178972062-0c15349b-7a42-4f48-b92b-8af5272277c2.png)

a small qol feature, you drag the bar between the difficulty numbers to move the whole difficulty range

so you can turn 3.0 to 4.0 into 2.3 to 3.3, 5.7 to 6.3 into 6.8 to 7.4, and 0.0 to 4.0 into 6.0 to 10.0

no limit counts as its own value but the lower end of the range shouldn't change if you're shifting to and from no limit

**Comment by peppy** [comment]
This was mentioned in the original feature as being harder to do. It will eventually be added, probably when the design is updated.

**Comment by Soetch** [comment]
Maybe like this ? We could next update values on the indicator like on the @iSimpFor's picture.

![image](https://user-images.githubusercontent.com/70801309/179238838-217d62d5-b743-4b70-a65a-009094bd6a71.png)

**Reply by frenzibyte** [reply]
For design, you probably just want to highlight both nubs and the line between them and move them around, no need for anything else.
